lib/dockerize_rails/constants.rb

```diff
@@ -58,7 +63,7 @@ module Constants
         aliases: %I[docker_stop stop],
         help: 'Stops Docker containers'.freeze,
         params: {
-            :'--delete' => 'also deletes the containers'.freeze
+            :'--delete' => 'Also deletes the containers'.freeze
```

Comment:
__[rubocop]__
* Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.

<!-- Comment by [SideCI](https://sideci.com) -->